docs/index.html

```diff
@@ -217,87 +219,164 @@ <h2>What is the purpose?</h2>
         ring: 0,
         label: "Java",
         website: "",
         active: true,
-        moved: 0
+        moved: 0,
+        description:`
+        <p>Java is an Object Oriented programming language. It is the 5th most popular language (behind Js, Html, SQL and Python) according to StackOverflow's most recent survey.</p>
+        <p>We use Java version 8 to run our backend application services, mainly using the Spring web framework.</p>
+        `
       },
       {
         quadrant: 3,
         ring: 0,
         label: "JavaScript",
         website: "",
         active: true,
-        moved: 0
+        moved: 0,
+        description:`
+        <p>JavaScript (JS) is an interpreted programming language and provides the client-side (browser) behaviour for modern web applications.</p>
+        <p>We use JS to power our Single Page Application (SPA) using both the angularjs and React frameworks.</p>
+        `
       },
       {
         quadrant: 3,
         ring: 0,
         label: "TypeScript",
         website: "",
         active: true,
-        moved: 0
+        moved: 0,
+        description:`
+        <p>TypeScript is a superset of JavaScript that adds static typing.</p>
+        <p>We use TypeScript to provide type safety to our JavaScript, helping us catch bugs quicker and write better, more readable code.</p>
+        `
       },
       {
         quadrant: 3,
         ring: 0,
-        label: "Final-Form",
+        label: "Final Form",
         website: "",
         active: true,
-        moved: 0
+        moved: 0,
+        description:`
+        <p>Final Form is a JavaScript form library.</p>
+        <p>In combination with React, Final Form powers the majority of the forms in Small Improvements. Final Form makes development of form based applications simpler and less error prone.</p>
+        `
       },
       {
         quadrant: 3,
         ring: 1,
-        label: "Node",
+        label: "Node.js",
         website: "",
         active: true,
-        moved: 0
+        moved: 0,
+        description:`
+        <p>Node.js (Node) is an open source JavaScript runtime environment that executes JavaScript outside of a browser.</p>
+        <p>Many of our internal tools, as well as our Cloud Functions run using Node.js. Node enables us to use the existing 
+          JavaScript knowledge in the team to great effect outside of the browser.</p>
+        `
       },
       {
         quadrant: 3,
         ring: 3,
         label: "Wicket",
         website: "",
         active: true,
-        moved: -1
+        moved: -1,
+        description:`
+        <p>Wicket is a component based web application framework for Java.</p>
+        <p>Used in the early days of Small Improvements, Wicket was a great way build fast and reliably. Wicket has since been replaced by the Spring framework
+          as our Single Page client-side application got increasingly larger and we required REST API's rather then full server-side rendering.</p>
```

Comment:
```suggestion
        <p>Used in the early days of Small Improvements, Wicket was a great way to build web applications fast and reliably. Wicket has since been replaced by the Spring framework
          as our Single Page client-side application got increasingly larger and we required REST APIs rather than full server-side rendering.</p>
```